hydroshare/settings.py

```diff
@@ -368,6 +369,11 @@
     "mezzanine.pages.middleware.PageMiddleware",
     "mezzanine.core.middleware.FetchFromCacheMiddleware",
     "hs_tracking.middleware.Tracking",
+    'security.middleware.XssProtectMiddleware',
```

Comment:
See the comment in the code about the ordering of the middlewares above and the documentation about the [CORS middleware setup](https://github.com/ottoyiu/django-cors-headers/#setup). Could these or the Cors Middleware safely be moved up the list?